src/browser/ui/ReactMount.js

```diff
@@ -518,10 +519,11 @@ var ReactMount = {
     // render but we still don't expect to be in a render call here.)
     warning(
       ReactCurrentOwner.current == null,
-      'unmountComponentAtNode(): Render methods should be a pure function of ' +
-      'props and state; triggering nested component updates from render is ' +
-      'not allowed. If necessary, trigger nested updates in ' +
-      'componentDidUpdate.'
+      '%s.unmountComponentAtNode(): Render methods should be a pure function ' +
+      'of props and state; triggering nested component updates from render ' +
+      'is not allowed. If necessary, trigger nested updates in ' +
+      'componentDidUpdate.',
+      this.constructor.displayName || 'ReactCompositeComponent'
```

Comment:
Actually, this message doesn't make sense. You'll only ever call `React.unmountComponentAtNode`, never a component itself. I think we actually want to leave the beginning of this warning in place and do something like "See the render method of %s." at the end.

Also, `this` isn't right in either of these cases. I believe `this` will refer to the global scope of the ReactMount module, not `this` inside of another component. I think you'll actually want to make use of `ReactCurrentOwner.current`.